### PR TITLE
Nicely integrated select box design

### PR DESCRIPTION
**Pull request by jancborchardt**
More integrated & less boxed in. As chatted about with @Henni :)
Please review @nextcloud/contacts 

Before:
screenshot from 2017-09-15 15-12-25
After:
screenshot from 2017-09-15 15-10-14

### CODE REVIEW

File: css/_details.scss [hunk 1]
```diff
@@ -273,6 +273,10 @@ detailsitem textarea,
 
 detailsitem select {
 	width: 100px;
+	border: none;
+	text-align: right;
+	opacity: .5;
+	color: #000;
```
Comment (skjnldsv) [inline css/_details.scss]: Please use server variables! :)

Comment (jancborchardt) [reply css/_details.scss]: Done, sorry! :D

File: css/_details.scss [hunk 2]
```diff
@@ -273,6 +273,16 @@ detailsitem textarea,
 
 detailsitem select {
 	width: 100px;
+	border: none;
+	text-align: right;
+	opacity: .5;
+	color: $color-main-text;
+	outline: none;
+}
+detailsitem select:hover,
```
Comment (skjnldsv) [inline css/_details.scss]: Please also add empty line before rules (travis will fail otherwise)

Comment (jancborchardt) [reply css/_details.scss]: Done too :) damn these old habits die hard

**Comment by jancborchardt** [comment]
@skjnldsv addressed the right align too

**Comment by jancborchardt** [comment]
Thanks for the thorough review @skjnldsv! Makes me feel rusty :D

**Comment by skjnldsv** [comment]
@jancborchardt Time to code more! :D

**Comment by jonatoni** [comment]
@skjnldsv you didn't let me enough time to review it 😛

**Pull request merged**